docs/oathkeeper/reference/configuration.mdx

```diff
@@ -6,5 +6,5 @@ title: Configuration
 ```mdx-code-block
 import ConfigMarkdown from '@site/src/components/ConfigMarkdown';
 
-<ConfigMarkdown src="https://raw.githubusercontent.com/ory/oathkeeper/master/.schema/config.schema.json" binary="oathkeeper" />
+<ConfigMarkdown src="https://raw.githubusercontent.com/ory/oathkeeper/master/.schemas/config.schema.json" binary="oathkeeper" />
```

Comment:
Actually, the correct file is now: https://github.com/ory/oathkeeper/blob/master/spec/config.schema.json